FTBFS with latest wlcs

```
/spread/mir/tests/acceptance-tests/wayland/miral_integration.cpp: In constructor â€˜{anonymous}::MirWlcsDisplayServer::MirWlcsDisplayServer()â€™:
/spread/mir/tests/acceptance-tests/wayland/miral_integration.cpp:1079:22: error: invalid conversion from â€˜const WlcsIntegrationDescriptor* (*)()â€™ to â€˜const WlcsIntegrationDescriptor* (*)(const WlcsDisplayServer*)â€™ [-fpermissive]
     get_descriptor = &::get_descriptor;
                      ^~~
```

--- a/tests/acceptance-tests/wayland/miral_integration.cpp
+++ b/tests/acceptance-tests/wayland/miral_integration.cpp
@@ -1062,7 +1062,7 @@ WlcsIntegrationDescriptor const descriptor {
     extensions
 };
 
-WlcsIntegrationDescriptor const* get_descriptor()
+WlcsIntegrationDescriptor const* get_descriptor(WlcsDisplayServer const* /*server*/)
 {
     return &descriptor;
 }

